zio-flow/shared/src/main/scala/zio/flow/ZFlowExecutor.scala

```diff
@@ -90,7 +90,7 @@ object ZFlowExecutor {
         state    <- stateRef.get
         vRef     <- ZIO.fromOption(state.getVariable(variableName))
         _        <- vRef.set(value.asInstanceOf)
-        _        <- stateRef.modify(state => (state.retry.forkDaemon, state.copy(retry = ZIO.unit))).flatten
+        //_ <- stateRef.modify(state => (state.retry.forkDaemon, state.copy(retry = ZIO.unit))).flatten
```

Comment:
Dead code?

Reply:
That's a miss. I will delete the commented line.

Reply:
The functions `getVariable` and `setVariable` in executor were not being used (Dead code). Deleted both.